raiden_contracts/constants.py

```diff
@@ -11,6 +13,57 @@
 CONTRACT_MONITORING_SERVICE = 'MonitoringService'
 CONTRACT_RAIDEN_SERVICE_BUNDLE = 'RaidenServiceBundle'
 
+# Deployed contract information
+# Deployed to Ropsten revival on 2018-09-03 from
+# raiden-contracts@fc1c79329a165c738fc55c3505cf801cc79872e4
+ROPSTEN_REGISTRY_ADDRESS = '0xf2a175A52Bd3c815eD7500c765bA19652AB89B30'
```

Comment:
Can we make this `ROPSTEN_TOKEN_NETWORK_REGISTRY_ADDRESS`?
Raiden client can rename it internally, but I wanted consistency with the contract names.

Reply:
yes